AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -821,7 +830,7 @@ - (void)transitionLayoutWithAnimation:(BOOL)animated
     return;
   }
   
-  [self invalidateCalculatedLayout];
```

Comment:
Above this line - it won't let me comment directly - CONDITION NEEDS UPDATE - since it only checks _calculated rather than also _pending layout to check if a constrainedSize is available.

Maybe we can call constrainedSizeForLayout there, and it might even return self.bounds.  But as long as it's not 0, 0 as a max size I think it should be safe to allow the method to proceed.